deploy/cr/minimal/backup.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: infinispan.org/v2alpha1
```

Comment:
Not sure that `minimal` folder good place for backup/restore CR's
It would be nice to have a separate one